docs/aws/eks-config.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+metadata:
+  name: ${CLUSTER_NAME}
+  region: ${AWS_DEFAULT_REGION}
+
+iam:
+  withOIDC: true
+  serviceAccounts:
+  - metadata:
+      name: karpenter
+      namespace: karpenter
+    attachPolicy:
+      Version: "2012-10-17"
+      Statement:
+      - Effect: Allow
+        Resource: "*"
+        Action:
+        # Write Operations
+        - "ec2:CreateLaunchTemplate"
+        - "ec2:CreateFleet"
+        - "ec2:RunInstances"
+        - "ec2:CreateTags"
+        - "iam:PassRole"
+        - "ec2:TerminateInstances"
+        # Read Operations
+        - "ec2:DescribeLaunchTemplates"
+        - "ec2:DescribeInstances"
+        - "ec2:DescribeSecurityGroups"
+        - "ec2:DescribeSubnets"
+        - "ec2:DescribeInstanceTypes"
+        - "ec2:DescribeInstanceTypeOfferings"
+        - "ec2:DescribeAvailabilityZones"
+        - "ssm:GetParameter"
+
+fargateProfiles:
+  - name: karpenter
```

Comment:
Might be better to separate into two fargate profiles, since kube-system shouldn't be named karpenter.